bjoern/server.c

```diff
@@ -213,8 +238,8 @@ ev_io_on_read(struct ev_loop* mainloop, ev_io* watcher, const int events)
     } else if(request->state.parse_finished) {
```

Comment:
I assume parse finished means we've received header and body? Could you add that to the comment below